### PR TITLE
fix: Node engine version upper range is <19 despite Node 19 support

### LINKED ISSUE #1731: Node engine version upper range is <19 instead of <20
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/Parse-SDK-JS/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the latest versions of [Parse Server](https://github.com/parse-community/parse-server/releases) and the [Parse JS SDK](https://github.com/parse-community/Parse-SDK-JS/releases). 

### Issue Description

Node engine version is <19 instead of <20

### Solution

The SDK already supports Node 19, and we forgot to bump this upper range in package.json.

### PR DESCRIPTION
**Pull request by mtrezza**
## Pull Request

- Report security issues [confidentially](https://github.com/parse-community/Parse-SDK-JS/security/policy).
- Any contribution is under this [license](https://github.com/parse-community/Parse-SDK-JS/blob/alpha/LICENSE).
- Link this pull request to an [issue](https://github.com/parse-community/Parse-SDK-JS/issues?q=is%3Aissue).

## Issue

Closes: #1731 

## Approach

Bump node engine upper range

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -127,7 +127,7 @@
     ]
   },
   "engines": {
-    "node": ">=14.21.0 <17 || >=18 <19"
```
Comment (dplewis) [inline package.json:130]: might need to do `npm install` to update the package-lock.json

**Review by dplewis — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (mtrezza) [reply package.json:130]: done thanks

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [4.0.0-alpha.11](https://github.com/parse-community/Parse-SDK-JS/releases/tag/4.0.0-alpha.11)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [4.1.0-beta.1](https://github.com/parse-community/Parse-SDK-JS/releases/tag/4.1.0-beta.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [4.1.0-alpha.1](https://github.com/parse-community/Parse-SDK-JS/releases/tag/4.1.0-alpha.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [4.1.0](https://github.com/parse-community/Parse-SDK-JS/releases/tag/4.1.0)

**Pull request merged**